2020-02-Spring/0_arrays_matrix/buy_stock/driver.java

```diff
@@ -0,0 +1,29 @@
+public class driver{
+    public static void main(String [] args){
+        int [] arr1 = {3};
+        int [] arr2 = {1,9};
+        int [] arr3 = {8,1};
+        int [] arr4 = {7,1,5,3,6,4};
+        int [] arr5 = {6,1,1,23,4,54,33,4,1,9,2,3,4,5,9,6,4,5,3,100};
+
+        Test("Max profit for arr1 is " + maxProfit(arr1), maxProfit(arr1) == 0);
+        Test("Max profit for arr2 is " + maxProfit(arr2), maxProfit(arr2) == 8);
+        Test("Max profit for arr3 is " + maxProfit(arr3), maxProfit(arr3) == 0);
+        Test("Max profit for arr4 is " + maxProfit(arr4), maxProfit(arr4) == 5);
+        Test("Max profit for arr5 is " + maxProfit(arr5), maxProfit(arr5) == 99);
```

Comment:
Need to call the method in our solution class

```suggestion
        Test("Max profit for arr1 is " + sol.maxProfit(arr1), sol.maxProfit(arr1) == 0);
        Test("Max profit for arr2 is " + sol.maxProfit(arr2), sol.maxProfit(arr2) == 8);
        Test("Max profit for arr3 is " + sol.maxProfit(arr3), sol.maxProfit(arr3) == 0);
        Test("Max profit for arr4 is " + sol.maxProfit(arr4), sol.maxProfit(arr4) == 5);
        Test("Max profit for arr5 is " + sol.maxProfit(arr5), sol.maxProfit(arr5) == 99);
```